src/main/kotlin/com/hedvig/lookupservice/graphql/types/InitiateDataCollectionInputV2.kt

```diff
@@ -0,0 +1,17 @@
+package com.hedvig.lookupservice.graphql.types
+
+import java.util.UUID
+
+sealed class InitiateDataCollectionInputV2
+
+data class DataCollectionPersonalNumberInput(
+    val reference: UUID,
+    val insuranceProvider: String,
+    val personalNumber: String
+) : InitiateDataCollectionInputV2()
+
+data class DataCollectionPhoneNumberInput(
+    val reference: UUID,
+    val insuranceProvider: String,
+    val phoneNumber: String
+) : InitiateDataCollectionInputV2()
```

Comment:
My suggestion would be to do somthing like this: 
```suggestion
sealed class InitiateDataCollectionInputV2 {
  abstract val reference: UUID
  abstract val insuranceProvider: String
}

data class DataCollectionPersonalNumberInput(
    override val reference: UUID,
    override val insuranceProvider: String,
    val personalNumber: String
) : InitiateDataCollectionInputV2()

data class DataCollectionPhoneNumberInput(
    override val reference: UUID,
    override val insuranceProvider: String,
    val phoneNumber: String
) : InitiateDataCollectionInputV2()
```